integrations-and-supported-tools/pytorch/scripts/helpers.py

```diff
@@ -0,0 +1,46 @@
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+from neptune.new.types import File
+import torch.nn.functional as F
+
+# Helper functions
+def save_model(model, name ="model.txt"):
+    print(f"Saving model arch as {name}.txt")
+    with open(f"{name}_arch.txt", "w") as f:  f.write(str(model))
+    print(f"Saving model weights as {name}.pth")
+    torch.save(model.state_dict(), f"./{name}.pth")
+    
+def get_obj_name(obj):
```

Comment:
we'll see what the guys on our product and dev say -> but I'd rather have the .__name__ than a function that people have to jump to see

Reply:
Great, let's see what they have to say.